fetch.bs

```diff
@@ -6710,6 +6748,362 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+       <a>navigable container</a>.
+
+       <li>
+        <p>If <var>otherContainer</var>'s <a>node document</a> and <var>otherDocument</var> do
+        not <a>share deferred-fetch quota</a>, then:
+
+        <ol>
+         <li><p>If <var>otherContainer</var>'s <a>subframe reserved deferred-fetch quota</a> is
+         <a>optional subframe deferred-fetch quota</a>, and <var>otherDocument</var> is
+         <a>allowed to use</a> the <a>policy-controlled feature</a>
+         {{PermissionPolicy/"deferred-fetch"}}, then increment <var>quota</var> by
+         <a>optional subframe deferred-fetch quota</a>.
+
+         <li><p>Otherwise, if <var>otherContainer</var>'s
+         <a>subframe reserved deferred-fetch quota</a> is
+         <a>minimal subframe deferred-fetch quota</a> and <var>otherDocument</var>
+         is <a>allowed to use</a> the <a>policy-controlled feature</a>
+         {{PermissionPolicy/"deferred-fetch-minimal"}}, then increment <var>quota</var> by
+         <a>minimal subframe deferred-fetch quota</a>.
+        </ol>
+      </ol>
+
+     <li>
+      <p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> of
+      <var>otherDocument</var>'s <a for=fetch>fetch group</a>'s
+      <a for="fetch group">deferred fetch records</a>:</p>
+      <ol>
+       <li><p>Let <var>requestLength</var> be the <a>total request length</a> of
+       <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>.
+
+       <li><p>Decrement <var>quota</var> by <var>requestLength</var>.
+
+       <li><p>If <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>'s
+       <a for=request>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>origin</var>,
+       then decrement <var>quotaForOrigin</var> by <var>requestLength</var>.
+      </ol>
+    </ol>
+
+   <li>
+    <p>Otherwise, if <var>otherNavigable</var> is not a <a for=/>top-level traversable</a>, and its
+    <a for=navigable>parent</a>'s <a>active document</a> and <var>document</var>
+    <a>share deferred-fetch quota</a>, then decrement <var>quota</var> by
+    <var>otherNavigable</var>'s <a>navigable container</a>'s
+    <a>subframe reserved deferred-fetch quota</a>.
+
+    <p class=note>This is a cross-origin/cross-agent subframe, so it spends some of the quota of
+    this document and its relatives.
+  </ol>
+ <li><p>If <var>quota</var> is less than zero, than return zero.
+
+ <li><p>If <var>quota</var> is less than <var>quotaForOrigin</var>, then
+ return <var>quota</var>.
+
+ <li><p>Return <var>quotaForOrigin</var>.
+</ol>
+</div>
+
+<div algorithm>
+<p>To <dfn export>reserve deferred-fetch quota</dfn> for a <a>navigable container</a>
+<var>container</var> given an <a for=/>origin</a> <var>originToNavigateTo</var>:
+
+<p class=note>This is called when the container document initiates a navigation. It potentially
+reserves either 64kb or 8kb of quota for the frame, if it doesn't <a>share deferred-fetch quota</a>
```

Comment:
```
It potentially reserves either 64kb or 8kb of quota for the frame, if it doesn’t share deferred-fetch quota with the container and the permissions policy allow
```

What are the arguments for `share deferred-fetch quota` here, i.e. which frame shares the same quota with the container?